### DISCUSSION TITLE
Can't build project

### DISCUSSION THREAD

**Discussion by Cthutq295** [hiperbou/kotlin-unreal#35] [Q&A] [answered]
@hiperbou Hello, i'm trying to get unreal engine work with kotlin. I move along with wiki and I stuck at the point where you have to clone project and then build it in order to open it in editor:
> You should be able to right click the file "KotlinUnreal.uproject" and select "Launch Game"
> You might be asked to recompile or update the project to your engine version and after that you should be seeing the First Person Game example.

I am using unreal engine 5.1 and have installed Unreal.js for the engine. When i try you launch "KotlinUnreal.uproject" i'm being told that project cannot be built and i have to do it manually
![image](https://github.com/hiperbou/kotlin-unreal/assets/10911451/4bdc6d44-f24e-4857-9b12-b1f4f74056e6)
![image](https://github.com/hiperbou/kotlin-unreal/assets/10911451/2d0a1a6e-854e-453c-be7f-8588bda080f2)
Opening solution with visual studio 2022 and trying to build project there I get another problem:
![image](https://github.com/hiperbou/kotlin-unreal/assets/10911451/ba2b1483-82a0-4a15-917b-ac82c75f517e)

So basically I can't launch the project at all. Tried to install 5.0.3 and it didn't help. What am I doing wrong? Any help with the problem?

**Comment by Cthutq295** [comment, marked as answer]
Managed to solve it with installing this: dotnet.microsoft.com/en-us/download/dotnet/3.1/runtime?cid=getdotnetcore&os=windows&arch=x64
